addons/mail/static/src/new/core/message_model.js

```diff
@@ -178,7 +178,7 @@ export class Message {
         if (!this.subject || !this.originThread || !this.originThread.name) {
             return false;
         }
-        const regexPrefix = /^((re|fw|fwd).?:.)*/gi;
+        const regexPrefix = /^((re|fw|fwd)\s*:\s*)*/i;
```

Comment:
This isn't really a big deal, as it won't affect the final behaviour, but if you want this regex to match a sequence of prefixes and nothing else, you should use the `+` quantifier on the outer capture group so as not to match ε.

```js
/^((re|fw|fwd)\s*:\s*)*/i.test(""); // true
/^((re|fw|fwd)\s*:\s*)+/i.test(""); // false